From 62520e9d0ab641b36fb770477792073e6aad4d54 Mon Sep 17 00:00:00 2001
From: Vadim Pasternak <vadimp@nvidia.com>
Date: Mon, 17 Jul 2023 16:24:58 +0000
Subject: [PATCH backport 6.1.42 47/85] dt-bindings: trivial-devices: Add
 infineon,xdpe1a2g7

Add new Infineon Multi-phase Digital VR Controller xdpe1a2g7

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>
---
 Documentation/devicetree/bindings/trivial-devices.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/trivial-devices.yaml b/Documentation/devicetree/bindings/trivial-devices.yaml
index 61746755c107..7637390dfe09 100644
--- a/Documentation/devicetree/bindings/trivial-devices.yaml
+++ b/Documentation/devicetree/bindings/trivial-devices.yaml
@@ -145,6 +145,8 @@ properties:
           - infineon,tlv493d-a1b6
             # Infineon Multi-phase Digital VR Controller xdpe11280
           - infineon,xdpe11280
+            # Infineon Multi-phase Digital VR Controller xdpe1a2g7
+            - infineon,xdpe1a2g7
             # Infineon Multi-phase Digital VR Controller xdpe12254
           - infineon,xdpe12254
             # Infineon Multi-phase Digital VR Controller xdpe12284
-- 
2.20.1

